guard/lib/guard/store/user.ex

```diff
@@ -254,27 +254,19 @@ defmodule Guard.Store.User do
           end
         end)
         |> Ecto.Multi.run(:delete_related_data, fn repo, _changes ->
-          from(o in Repo.Organization, where: o.creator_id == ^user_id)
-          |> repo.all()
-          |> Enum.each(fn org ->
-            :ok = Guard.Api.Project.destroy_all_projects_by_org_id(org.id)
+          repo.delete_all(from(r in Repo.RepoHostAccount, where: r.user_id == ^user_id))
 
-            repo.delete_all(from(m in Repo.Member, where: m.organization_id == ^org.id))
-
-            repo.delete_all(
-              from(s in Repo.OrganizationSuspension, where: s.organization_id == ^org.id)
+          members_query =
+            from(m in Guard.FrontRepo.Member,
+              join: rha in Guard.FrontRepo.RepoHostAccount,
```

Comment:
Will it have anything to join with if we delete all the repo_host_accounts on the line 257?